nox/tox_to_nox.py

```diff
@@ -21,20 +21,31 @@
 from typing import Any, Iterator
 
 import jinja2
-import tox.config
+from tox import __version__ as TOX_VERSION
 
-_TEMPLATE = jinja2.Template(
-    pkgutil.get_data(__name__, "tox_to_nox.jinja2").decode("utf-8"),  # type: ignore[union-attr]
-    extensions=["jinja2.ext.do"],
-)
+TOX3 = TOX_VERSION[0] == "3"
+
+if TOX3:
```

Comment:
I think it's better to check for tox 4 here and flip the conditional? That way we only do the tox 4 logic for tox 4, everything else goes through the previous implementation

Reply:
Done